apps/core/components/ItineraryCard/ItineraryCardRow.js

```diff
@@ -17,5 +17,9 @@ const styles = StyleSheet.create({
   container: {
     paddingHorizontal: 15,
     paddingVertical: 5,
+    web: {
+      paddingHorizontal: 0,
```

Comment:
`padding:0` instead?

Reply:
I've tried it but it will not work. The values needs to be overridden directly like this.
I encounter same situation for example even in `ItineraryCard` with `borderBottomWidth` where setting global `borderWidth` to reset border wasn't enough.